src/System.Reflection.Emit/tests/TypeBuilder/TypeBuilderDefineDefaultConstructor.cs

```diff
@@ -46,85 +46,85 @@ public TypeBuilder RetriveTestTypeBuilder(TypeAttributes typeAtt)
         }
 
         [Fact]
-        public void PosTest1()
+        public void TestDefaultConstructor1()
         {
             ExePosTest(TypeAttributes.Public | TypeAttributes.Class, MethodAttributes.Public, BindingFlags.Public | BindingFlags.Instance);
         }
 
         [Fact]
-        public void PosTest2()
+        public void TestDefaultConstructor2()
```

Comment:
I thought by looking at the arguments passed over to ExePosTest it was evident that we are testing the different combo of MethodAttributes and BindingFlags. I couldn't think of better names.